test/integration/include_custom_model.sdf

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<sdf version="1.7">
```

Comment:
To make this a valid xml, I think we need to define the `experimental` namespace. My suggestion would be to use `http://sdformat.org/schemas/experimental`.
```suggestion
<sdf version="1.7" xmlns:experimental="http://sdformat.org/schemas/experimental">
```

Reply:
Done 5c167b32c5d79d57292702ebfb4114b39e4506cb